bundles/org.openhab.binding.solax/src/main/resources/OH-INF/i18n/solax.properties

```diff
@@ -111,6 +115,10 @@ channel-type.solax.last-retrieve-time-stamp.label = Last Retrieve Time Stamp
 channel-type.solax.last-retrieve-time-stamp.description = Last time with a successful retrieval of data
 channel-type.solax.raw-data-type.label = Raw Data
 channel-type.solax.raw-data-type.description = The raw JSON data retrieved from the inverter's Wi-Fi module.
+channel-type.solax.inverter-temperature.label = Inverter Temperature
+channel-type.solax.inverter-temperature.description = Inverter Temperature
+channel-type.solax.inverter-workmode-code.label = Inverter Workmode Code
```

Comment:
the workmode (non code) channel seems t o be missing. Can you re-run i18n plugin?